engine/src/main/java/com/datatorrent/stram/client/StramAgent.java

```diff
@@ -417,11 +417,25 @@ private StramWebServicesInfo retrieveWebServicesInfo(String appId)
       String appPath = response.getString("appPath");
       String user = response.getString("user");
       JSONObject permissionsInfo = null;
-      try (FSDataInputStream is = fileSystem.open(new Path(appPath, "permissions.json"))) {
-        permissionsInfo = new JSONObject(IOUtils.toString(is));
+      Path permissionsPath = new Path(appPath, "permissions.json");
+      LOG.debug("Checking for permission information in file {}", permissionsPath);
+      try {
+        if (fileSystem.exists(permissionsPath)) {
+          LOG.info("Loading permission information");
+          try (FSDataInputStream is = fileSystem.open(permissionsPath)) {
+            permissionsInfo = new JSONObject(IOUtils.toString(is));
+          }
```

Comment:
LOG.debug() that permissions were loaded successfully? The overall amount of logging for the feature that is rarely used seems quite excessive.